external/kafka-0-10-sql/pom.xml

```diff
@@ -79,6 +79,10 @@
       <artifactId>kafka-clients</artifactId>
       <version>${kafka.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
```

Comment:
We may need a new entry in licenses/ and LICENSE for this, if it's not there already. Likewise for JAXB.
You'll want to run the script in dev/ to update the deps/ files too.

Reply:
Thanks. Yes will do after making all tests pass.

Reply:
Should we add the license before we merge it?

Reply:
@srowen @dbtsai actually this was [already added](https://github.com/apache/spark/blob/master/LICENSE-binary#L322) - it makes sense because we're already using this from the old hadoop transitive dependencies, what's new is that we're making it explicit since hadoop-client-api excludes it.